### ISSUE TITLE
Should distributionGroupId be destinationGroupIds?

### ISSUE THREAD

**Issue by jessedc** [MicrosoftDocs/vsts-docs#4691]
Looking through the source code, should the documentation mention destinationGroupIds rather than distributionId when the destination type is Groups?

https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/AppCenterDistributeV3/appcenterdistribute.ts#L511

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: da6643ac-ea32-1761-a7e9-0f17a3c0b8ee
* Version Independent ID: 2442af0f-3f01-a91c-a98a-29c2f32d35b9
* Content: [App Center Distribute task - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/app-center-distribute?view=azure-devops#feedback)
* Content Source: [docs/pipelines/tasks/deploy/app-center-distribute.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/tasks/deploy/app-center-distribute.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @DavidStaheli
* Microsoft Alias: **dastahel**

**Comment by DRAirey1** [comment]
This feature 'destinationGroupIds' isn't working. No matter what value is used in this setting, the app is always distributed to the default 'Collaborators' group. To whom should I report the issue?

**Comment by DRAirey1** [comment]
The documentation is wong!!! I spent hours on this problem. The devOps task AppCenterDistribute@3 would not drop the application in the specified group. No matter what I did, it would drop the release in the default (collaborators) group. It wasn't until I came across this case and the link to the source code that I figured it out. The documentation [()](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/app-center-distribute-v3?view=azure-pipelines) states:

```
distributionGroupId - Destination IDs
Input alias: destinationGroupIds. string. Optional. Use when destinationType = groups.

The IDs of the distribution groups who will receive the build release. Leave it empty to use the default group, and use commas or semicolons to separate multiple IDs.
```
But if you look at the source code, the actual name of the property is: **destinationGroupIds** Emphasis on the 's'.
